### ISSUE TITLE
Multi-label classification ?

### ISSUE THREAD

**Issue by zeakey** [dmlc/mxnet#1758]
Can I implement Multi-Label classification by using mxnet's LogisticRegressionOutput as loss function ?
I mean, in other words, does LogisticRegressionOutput support probability vector as input rather than a singular ?

**Comment by piiswrong** [comment]
Yes it does.
Checkout https://github.com/dmlc/mxnet/blob/master/example/kaggle-ndsb2/Train.py

**Comment by zeakey** [comment]
@piiswrong ok, Thank you very much !

**Comment by yupbank** [comment]
i think this is a multi-class regression example? in multi-label regression, there is no softmax constrain across label dimensions.

**Comment by miraclewkf** [comment]
There is an example of multi-label image classification: https://github.com/miraclewkf/multilabel-MXNet, which use sigmoid instead of LogisticRegressionOutput and needn't recompile mxnet.
